src/WebJobs.Extensions.CosmosDB/Config/CosmosDBExtensionConfigProvider.cs

```diff
@@ -153,6 +154,8 @@ internal static bool IsSupportedEnumerable(Type type)
             return false;
         }
 
+        internal static string BuildCacheKey(string connectionString, string preferredLocations, bool useMultipleWriteLocations) => $"{useMultipleWriteLocations}{preferredLocations}{connectionString}";
```

Comment:
nit -- can we use a `|` (or something else) as a delimiter between the values? It'd be easier to parse if we need to debug this.

Reply:
Is there a reason that they are written in reverse-order? I'd assume `connectionString` would be first to sort more logically... although it probably doesn't really matter.

Reply:
Addressed 😄